Assets/Scripts/NPCs/StateMachine/States/Chase.cs

```diff
@@ -31,21 +38,25 @@ private void Chasing()
 
     private Vector3 ComputeChaseVelocity()
     {
-        Vector3 goalLocation;
-        
+        Vector3 targetLocation;
+
         //The NPC will predict the future position of the player (pursuit behavior)
-        if (_npcManager.GetDistanceWithPlayer() > 1)
+        if (_npcManager.GetDistanceWithPlayer() > NPCsGlobalVariables.ChasePlayerRange) // TODO add raycast here
```

Comment:
I think we need a raycast to see if we can directly go to the player instead of only checking the distance (line of sight checking is better than just the distance because it account for wall and stuff)

Check Node.CanSeeNode function for an exemple accounting for the size of the NPC while checking the line of sight

Reply:
Look at my answer above.

Reply:
At the moment, there seems to be issues with `Node.CanSeeNode` and it always returns false. We're not sure why. So we will remove the calls that were added in the last commit for the moment and log a bug or something to address this afterwards.

Reply:
yeah i just realized that, i'm going to fix it in another branch, sry for the inconvenience